### PR TITLE
Tlunizin 683

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

**Comment by lsloan** [comment]
When you post a code review card on Trello, use the shorter URL for the GH PR.  When the shorter one is used, Trello will recognize it as a PR and it will decorate the link with a GH icon and add the title from the PR (If one is given, which it should be so that PRs can be distinguished from each other.) to the link in the card.  That is, don't use this in Trello:

`https://github.com/tl-its-umich-edu/canvas_courses_report/pull/8/files`

Use this instead:

`https://github.com/tl-its-umich-edu/canvas_courses_report/pull/8`

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+<code>-DcanvasCourseReportPath=file:/file-path/coursereport.properties</code><br/>
+
+
+4. Add the following 5 properties to coursereport.properties: 
+
+    <code>canvas.admin.token=canvas token  
+    canvas.url=target canvas server e.g. https://umich.test.instructure.com  
+    use.test.url=true  
+    ldap.server.url=ldap server e.g. ldap://ldap.itd.umich.edu:389/dc=umich,dc=edu  
+    mcomm.group=group that can use this tool e.g. its-canvas-reports</code>
```
Comment (lsloan) [inline README.md]: The formatting for this block of code is wrong.  See the GH rendering of this document to see how it looks:

https://github.com/tl-its-umich-edu/canvas_courses_report/blob/TLUNIZIN-683/README.md

Use the same kind of formatting that was used for steps six or seven, below, which appear to be correct.  In general, avoid the HTML-like tags in Markdown documents.  Use the accent mark instead.

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+
+
+5. Run this in local
```
Comment (lsloan) [inline README.md]: "in local" -> "on localhost", "on your local machine", etc.?

File: README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+    echo "$JOB_NAME | Build: $BUILD_NUMBER | $GIT_URL | $GIT_COMMIT | $GIT_BRANCH | $BUILD_ID" >> build.txt
+ ```
+
+## Notes
+
+If <code>use.test.url</code> is true, users will be able to execute the tool as if authenticated as the user specified in the URL parameter <code>?testUser=uniqname</code>. In Production this variable is  false. Based on this property property testUser is not allowed in Production.
+
+ldap is used for authorizing the user and he needs to be part of particular Mcommunity group to be authorized to use the tool.
+
+## Know issues
```
Comment (lsloan) [inline README.md]: "Know" -> "Known"

File: README.md [hunk 4]
```diff
@@ -1 +1,74 @@
...
+1)Currently in application each request for generating a course report is placed on separate thread and all the list of threads created likewise are placed in memory(java MAP object). The application is  deployed to tlsupportqa/tlsupport application servers and all request are directed only to one webserver( e.g., tomcat). Using a map in memory will work fine as all the requests go back to the same web server. if we ever have multiple webservers, 
+   a) one solution is to make the Load balancer session "sticky" so that the request always goes back to same server. 
+   b) Based on the usage we make have to revisit this issue to have better solution to store the list of the Threads in Database or Session.
+
```
Comment (lsloan) [inline README.md]: The section for "Known Issues" is not formatted correctly.  MD expects items in numbered lists to be prefixed with "1. " (one, period, space).  Nested list items can be indented, but they should be numbered in the same way.  By default, the nested items will automatically be shown with lowercase Roman numerals.  E.g.:

``` text
1. Currently...
  1. one solution...
  1. Based on...
```

Which will appear as:
1. Currently...
   1. one solution...
   2. Based on...

File: README.md [hunk 5]
```diff
@@ -1 +1,74 @@
...
+
+
+
+
+
+ 
+
```
Comment (lsloan) [inline README.md]: Delete unnecessary trailing blank lines.

File: src/main/webapp/build.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+This is build.txt.  See README.txt for information about a build script to fill in this file.
```
Comment (lsloan) [inline src/main/webapp/build.txt]: "README.txt" -> "README.md"

File: README.md (hunk 1 above)
Comment (pushyamig) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (pushyamig) [inline README.md]: Done

File: README.md (hunk 3 above)
Comment (pushyamig) [inline README.md]: Done

File: README.md (hunk 4 above)
Comment (pushyamig) [inline README.md]: Done.

File: src/main/webapp/build.txt (hunk 6 above)
Comment (pushyamig) [inline src/main/webapp/build.txt]: Done

File: README.md (hunk 5 above)
Comment (pushyamig) [inline README.md]: Done

**Pull request merged**